tests/openvino/tools/calibrate.py

```diff
@@ -612,16 +689,26 @@ def transform_fn(data_item):
         return input_data
 
     calibration_dataset = nncf.Dataset(model_evaluator.dataset, transform_fn)
+
+    if get_allow_reshape_input(accuracy_checker_config):
+        ov_model = maybe_reshape_model(
+            ov_model,
+            calibration_dataset,
+            quantization_parameters.get("subset_size", 300),
+            model_evaluator.launcher.input_to_tensor_name,
+        )
+        model_evaluator.load_network([{"model": ov_model}])
```

Comment:
Is `model_evaluator.load_network([{"model": ov_model}])` mandatory?

Reply:
Yes, it's needed to notify AC that model has new shape (static or dynamic). Skipping this leads to wrong dataset reshaping as AC prepares dataset based on loaded model inputs

Reply:
Yes, it's needed to notify AC that model has new shape (static or dynamic). Skipping this leads to wrong dataset reshaping as AC prepares dataset based on loaded model inputs